benchmark/scripts/test_compare_perf_tests.py

```diff
@@ -507,6 +507,13 @@ class OldAndNewLog(unittest.TestCase):
                              for line in
                              new_log_content.splitlines()])])
 
+    old_results['D'] = PerformanceTestResult(
```

Comment:
This is a visualization of an example of the problematic benchmark used in these unit tests:
![Dubious Benchmark](https://user-images.githubusercontent.com/1046899/64974128-86db3c00-d8ac-11e9-8b44-1593e5b24e97.png)

Reply:
These are two independent executions, picked to showcase the rarely occurring but very significant variation present in one of them.